lib/ecto/query/inspect.ex

```diff
@@ -82,6 +82,13 @@ defimpl Inspect, for: Ecto.Query do
   defp unbound_from(%Ecto.Query{} = query) do
     "^" <> inspect(query)
   end
+  defp unbound_from(%Ecto.Query.FromExpr{} = from) do
+    if is_nil(from.prefix) do
+      unbound_from(from.source)
+    else
+      "FromExpr(\"#{from.prefix}\", #{unbound_from(from.source)})"
```

Comment:
TODO: we should emit valid queries as much as possible. So we need to figure out how we are going to support from prefixes in the keyword query.

Reply:
`"prefix"."table"` is how Postgres has this feature specified. Do you know how other systems make this syntax?

Reply:
This is inspecting in the Elixir syntax. So we need to reproduce the same query syntax as the developer would write when building the query.